src/assets/main.css

```
@@ -142,6 +124,21 @@ input{font-family:var(--font);outline:none}
 .delay-4{animation-delay:0.12s;opacity:0}
 .delay-5{animation-delay:0.15s;opacity:0}
 
+/* ── Skeleton ── */
+@keyframes skeletonShimmer{
```

Stylelint (17.4.0)

[error] Expected keyframe name "skeletonShimmer" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)